pkg/config/engine/containerd/option.go

```diff
@@ -78,3 +78,14 @@ func WithContainerAnnotations(containerAnnotations ...string) Option {
 		b.containerAnnotations = containerAnnotations
 	}
 }
+
+// CommandLineSource returns the CLI-based containerd config loader
+func CommandLineSource(hostRoot string) toml.Loader {
+	var cliArgs []string
+	if hostRoot != "" {
+		cliArgs = append(cliArgs, "chroot", hostRoot)
+	}
+	cliArgs = append(cliArgs, "containerd", "config", "dump")
+
+	return toml.FromCommandLine(cliArgs[0], cliArgs[1:]...)
+}
```

Comment:
Let's rewrite this as:
```suggestion
// CommandLineSource returns the CLI-based containerd config loader
func CommandLineSource(hostRoot string) toml.Loader {
	commandLine := chrootIfRequired(hostRoot, "containerd", "config", "dump")
	return toml.FromCommandLine(commandLine[0], commandLine[1:]...)
}

func chrootIfRequired(hostRoot string, commandLine ...string) []string {
	if hostRoot == "" || hostRoot == "/" {
		return commandLine
	}

	return append([]string{"chroot", hostRoot}, commandLine...)
}
```

Reply:
Done, thanks